quickstarts/microsoft.dbformysql/create-flexible-server-public-access/azuredeploy.parameters.json

```diff
@@ -0,0 +1,73 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "administratorLogin": {
+            "value": "GEN-UNIQUE"
+        },
+        "administratorLoginPassword": {
+            "value": "GEN-PASSWORD"
+        },
+        "location": {
+            "value": "eastus"
+        },
+        "serverName": {
+            "value": "GEN-UNIQUE"
+        },
+        "serverEdition": {
+            "value": "Burstable"
+        },
+        "skuName": {
+            "value": "Standard_B1ms"
+        },
+        "storageSizeGB": {
+            "value": 32
+        },
+        "storageIops": {
+            "value": 360
+        },
+        "storageAutogrow": {
+            "value": "Enabled"
+        },
+        "availabilityZone": {
+            "value": ""
+        },
+        "version": {
+            "value": "5.7"
+        },
+        "tags": {
+            "value": {}
+        },
+        "haEnabled": {
+            "value": "Disabled"
+        },
+        "standbyAvailabilityZone": {
+            "value": ""
+        },
+        "firewallRules": {
+            "value": {
+                "rules": [
+                    {
+                        "name" : "AllowMyIP",
+                        "startIPAddress" : "59.97.41.110",
```

Comment:
What are these ip addresses?

Reply:
Sample IP addresses. Will replace them with generic values.

Reply:
use something that will make it obvious they are "placeholder" and not expose the resource to an unknown IP... depends on what you support, could be 0.0.0.0, 10. space, 169. etc